# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome14/libglade10/libglade-10.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./glade/glade-sax.c.orig	Mon Jun 18 12:36:11 2001
+++ ./glade/glade-sax.c	Mon Jun 18 12:36:21 2001
@@ -20,6 +20,8 @@
  * Boston, MA  02111-1307, USA.
  */
 
+typedef char CHAR;
+
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
